assets/configs/tenancy.php

```diff
@@ -184,6 +184,16 @@
          */
         'password-generator' => Hyn\Tenancy\Generators\Database\DefaultPasswordGenerator::class,
 
+        /**
+         * By default tenancy will automatically create a database user with access
+         * to the tenant database. By disabling this feature you are able to
+         * implement your own user generation or - even if not recommended -
+         * re-use existing credentials.
```

Comment:
@Thijmen I've rewritten this comment. You mentioned it uses the tenant connection, but in fact it clones the system connection config, not the tenant one? If so it should read:

"re-use existing system connection credentials"